common/client.go

```diff
@@ -271,7 +277,6 @@ func (c *DatabricksClient) Authenticate(ctx context.Context) error {
 		log.Printf("[INFO] Configured %s auth: %s", auth.name, c.configDebugString()) // lgtm[go/clear-text-logging]
 		c.authVisitor = authorizer
 		c.AuthType = auth.name
```

Comment:
```suggestion
		c.AuthType = auth.name
		c.fixHost()
```

can we fix it before and after the auth resolution? e.g. misconfigured databrickscfg